brianlong/www.validators.app
# 182427127-associate-stake-accounts-to-new-validators

#### What's this PR do?
- stake logic should select active vote_accounts if available

#### How should this be manually tested?
- run tests
- check example from ticket on staging

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182427127)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
